stages/org.osbuild.ostree.post-copy

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+"""Apply post-copy updates to an ostree repo/deployment
+
+The way osbuild works the ostree deployment is built in a chroot and
+stored as a regular directory of files before finally being copied to
+the physical filesystem. This means that for example, ostree fs-verity
+support doesn't work, as the fs-verity setting of files is not copied.
+
+To support fs-verity in generated images you have to run this stage
+after copying the final ostree tree onto the target filesystem.
+
+Notes:
+ - Ensure the target filesystem supports fs-verity. See e.g. the
+   `verity` option in org.osbuild.mkfs.ext4.
+ - Requires ostree version 2023.8 or later in the buildroot.
+"""
+
+import os
+import sys
+
+import osbuild.api
+from osbuild.util import ostree
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "properties": {
+     "sysroot": {
+       "type": "string",
+      "description": "Custom sysroot path",
+      "pattern": "^\\/(?!\\.\\.)((?!\\/\\.\\.\\/).)+$"
+    }
+  }
+},
+"devices": {
```

Comment:
Sorry, I think I overlooked this; but we aren't using devices anywhere in this stage are we?

Reply:
Not directly, but the mounts don't work without devices that have the filesystems.
In sample-images for example, this is typically used like so:
```
type: org.osbuild.ostree.post-copy
devices:
  root:
    type: org.osbuild.loopback
    options:
      filename: disk.img
mounts:
- name: root
  type: org.osbuild.ext4
  source: root
  target: /
```

Reply:
Thanks that makes sense.